lib/response_bank.rb

```diff
@@ -59,7 +59,7 @@ def cache_key_for(data)
       when Time, DateTime
         data.to_i
       when Date
-        data.to_time.to_i
+        data.to_s # Date#to_i does not support timezones, using iso8601 instead
```

Comment:
What do you mean by "does not support timezones"?

```ruby
>> Date.today.to_s
=> "2023-03-27"
```

Date#to_s doesn't include it either, nor `Time#to_i` etc. So it's not very clear to me what issue this is solving.